### ISSUE TITLE
Several modded wood types don't look right when applies to cogwheels

### ISSUE THREAD

**Issue by saltyseadoggo** [Rabbitminers/Extended-Cogs#68]
**Describe the bug**
Several modded wood types don't look right when you apply them to cogwheels. Cogwheels with these wood types applied use oak log/plank textures instead of the given wood type's textures.
The wood types I found have this issue are as follows:

[BYG](https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go)
- Embur (`byg:embur_planks`)
- Imparius (`byg:imparius_planks`)

[Regions Unexplored](https://www.curseforge.com/minecraft/mc-mods/regions-unexplored)
- Alpha (`regions_unexplored:alpha_planks`)
- Sculkwood (`regions_unexplored:sculkwood_planks`)
- Painted planks in all 16 vanilla dye colors (`regions_unexplored:`\<color\>`_painted_planks`)

[Ars Nouveau](https://www.curseforge.com/minecraft/mc-mods/ars-nouveau)
- Archwood (`ars_nouveau:archwood_planks`)

The common thread with these wood types seems to be that they either lack a stripped log/stem, or the stripped log/stem has an id with an unexpected format.

**To Reproduce**
1: Start the game with Create, C:Extended Cogwheels and any of the above mods loaded.
2: Place any cogwheel.
3: Obtain any of the plank items mentioned above.
4: Right click the cog while holding the plank.

**Expected behavior**
The cogwheel would take on an appearance matching the plank I used on it, as it would for all other wood types.

**Screenshots**
The errored archwood cog. All of the cogs with this issue look like this.
![2023-09-21_21 47 46](https://github.com/Rabbitminers/Extended-Cogwheels/assets/69029346/451b5af9-2b4c-47b2-a742-728aada556e7)

**Loader & Other Mods:**
 - Forge 43.2.21
 - Create 0.5.1c
 - [BYG](https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go) 2.0.1.4
 - [CorgiLib](https://www.curseforge.com/minecraft/mc-mods/corgilib) 1.0.0.34
 - [GeckoLib](https://www.curseforge.com/minecraft/mc-mods/geckolib/files/4407241) 3.1.40
 - [Regions Unexplored](https://www.curseforge.com/minecraft/mc-mods/regions-unexplored) 0.4.1_1
 - [Ars Nouveau](https://www.curseforge.com/minecraft/mc-mods/ars-nouveau) 3.17.10
 - [Curios API](https://www.curseforge.com/minecraft/mc-mods/curios) 5.1.4.1

**Comment by StormDragon-64** [comment]
Ecologics flowering acacia planks also have this issue. Create’s waterwheels display these wood types just fine, and this issue affects all cogwheel types (large, shaftless, etc.) btw.

**Comment by Rabbitminers** [comment]
> The common thread with these wood types seems to be that they either lack a stripped log/stem, or the stripped log/stem has an id with an unexpected format.

Yep that's exactly it, we use the stripped texture of a log as it tends to be the smoothest. There's not much we can do about it if that texture can't be found. Unless you have any suggestions we could either leave it as it is and default the textures, pop up a message saying the wood types can't be used or try and fall back on a different texture, say the plank

**Comment by saltyseadoggo** [comment]
i do have a suggestion, actually: perhaps add a config where end users can manually specify what textures to use for a cog that has had a given plank type applied
here's an example to show what the formatting could look like
```json
{
  "regions_unexplored:sculkwood_planks": {
    "top": "regions_unexplored:sculkwood_log_dark_top",
    "side": "regions_unexplored:sculkwood_log_dark"
  }
}
```

**Comment by Rabbitminers** [comment]
Interesting, we could definitely do something like that although it would probably work better being in data packs rather than in config

**Comment by saltyseadoggo** [comment]
sounds good to me~

**Comment by Rabbitminers** [comment]
Can you send me the ids of the stripped logs you would expect to be used in these cases

**Comment by saltyseadoggo** [comment]
sure~

BYG
- Embur: `byg:stripped_embur_pedu`
- Imparius doesn't have a stripped variant; you can either default to the planks or use the vaguely similar `byg:fungal_imparius_block`, though note that it uses the `cube_all` model and not the log model

Regions Unexplored
- Alpha: `minecraft:stripped_oak_log`
- Sculkwood: `regions_unexplored:sculkwood_log_dark`
- The 16 painted planks don't have corresponding logs, so i say just default to the plank texture

Ars Nouveau
- Archwood: `ars_nouveau:stripped_purple_archwood_log`
